src/lambda/models/samLambdaRuntime.ts

```diff
@@ -84,9 +83,23 @@ export function getFamily(runtime: string | undefined): SamLambdaRuntimeFamily {
         case 'java':
             return SamLambdaRuntimeFamily.Java
         case 'ruby2.5':
+        case 'ruby':
             return SamLambdaRuntimeFamily.Ruby
         default:
             throw new Error(`Unrecognized runtime: '${runtime}'`)
 
     }
 }
+
+export function isInterpretedRuntime(runtime: string): boolean {
+    switch (getFamily(runtime)) {
+        case SamLambdaRuntimeFamily.DotNet:
+        case SamLambdaRuntimeFamily.Java:
+        case SamLambdaRuntimeFamily.Go:
+            return false
+        case SamLambdaRuntimeFamily.NodeJS:
+        case SamLambdaRuntimeFamily.Python:
+        case SamLambdaRuntimeFamily.Ruby:
+            return true
+    }
+}
```

Comment:
Add a fallback to handle anything not caught by the switch statement.
Log, and either return false or throw an Error.

Reply:
Would the error thrown by `getFamily()` (in the `switch` statement) not bubble up?

Reply:
Good point. It would. However, an entry could get added to `SamLambdaRuntimeFamily` that isn't added here.
What if we make the known interpreted families return true, and add the `default:` case to the set returning false.

Reply:
It might be a better idea to change this from a boolean to an enum (with an unknown value). Do we want to rely on this returning interpreted vs. compiled, or interpreted vs. not-interpreted?

If the latter is the case, I'll throw an error in the default statement.

Reply:
I don't see the value in making an enum here, it doesn't seem like a strong use case at this point.
My preference is to keep the bool behavior, and throw "unknown" in the default case.

Alternatively, it may be worth scrapping this method, and making a `shouldAppendRelativePathToFunctionHandler(runtime)` method in the same file as `getRelativeFunctionHandler`, since that is the problem we're trying to solve today.

Reply:
✔️